tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py

```diff
@@ -235,19 +213,40 @@ def test_bulk_pod_attach_performance(self, teardown_factory, bulk_size):
         )
 
         # Write the test results into the ES server
-        full_results.es_write()
-        # write the ES link to the test results in the test log.
-        log.info(f"The result can be found at : {full_results.results_link()}")
+        if full_results.es_write():
+            res_link = full_results.results_link()
+            # write the ES link to the test results in the test log.
+            log.info(f"The result can be found at : {res_link}")
+
+            # Create text file with results of all subtest (4 - according to the parameters)
+            self.write_result_to_file(res_link)
+
+    def test_bulk_pod_attach_results(self):
+        """
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (4)
+        """
+
+        self.number_of_tests = 4
+        self.results_path = get_full_test_logs_path(
+            cname=self, fname="test_bulk_pod_attach_performance"
+        )
+        self.results_file = os.path.join(self.results_path, "all_results.txt")
+        log.info(f"Check results in {self.results_file}")
+
+        self.check_tests_results()
+
+        self.push_to_dashboard(test_name="Bulk Pod Attach Time")
```

Comment:
@Avilir Are we planning to use similar functionality in all tests ? to push result to dashboard ?, If so I would suggest to write this as a function or some other way to not duplicate this code again.

Reply:
@ramkiperiy, yes we are planning to do it for all tests, this function must be a part of the test, since it collect results from all previous tests in the same .py file and can not run from outside the file of as part of individual test (unless there are not any parametrize for the test - which we do not have this scenario)

Reply:
So in-case of failed tests, results will not be updated only for those tests ? as per the log dir will be created only for tests which passed. Am I right here ?

Reply:
in case of failed test, no results will be pushed into the dashboard, since we want complete set of results from single run on the same cluster we must have that all tests (of a single function) will pushed together.